src/main/kotlin/com/virtuslab/dokka/site/parser.kt

```diff
@@ -117,7 +117,7 @@ open class ExtendableMarkdownParser(private val text: String, private val extern
                     URL(link)
                     null
                 } catch (e: MalformedURLException) {
-                    externalDir(link)
+                    externalDri(link)
```

Comment:
Why?

Reply:
unrel fix (note the val return type in particular `private val externalDri: (String) -> DRI?`)